clang/docs/ReleaseNotes.rst

```diff
@@ -544,6 +548,14 @@ Improvements to Clang's diagnostics
 - Clang no longer emits a "declared here" note for a builtin function that has no declaration in source.
   Fixes #GH93369.
 
+- On ARM, Clang no longer suggests adding ``__attribute__((interrupt))`` to
```

Comment:
I would phrase this "For the ARM target", since Clang could be running on any host architecture.